cardano-db/src/Cardano/Db/Multiplex.hs

```diff
@@ -105,10 +105,8 @@ runExtraMigrations trce blockNoDiff PruneConsumeMigration {..} = do
           ExtraCons.migrateTxOut (Just trce)
         (False, _, True) -> do
           shouldInsertToMigrationTable
-          liftIO $ logInfo trce "Running extra migrations consumed_tx_out and prune tx_out"
-          ExtraCons.migrateTxOut (Just trce)
           liftIO $ logInfo trce "Now Running extra migration prune tx_out"
-          ExtraCons.deleteConsumedTxOut trce blockNoDiff
+          ExtraCons.deleteAndUpdateConsumedTxOut trce hasConsumedField blockNoDiff
```

Comment:
`hasConsumedField` is always False here, so why it gets passed to `deleteAndUpdateConsumedTxOut`?

Reply:
having a `TVar` would do the trick but I don't have access to `SyncEnv` here in `/cardano-db/`. This logic should all probably live in `/cardano-db-sync/` 🤔